services/blockchain-indexer/tests/unit/shared/dataservice/business/interoperability/channel.test.js

```diff
@@ -0,0 +1,150 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+/* eslint-disable import/no-dynamic-require */
+const { resolve } = require('path');
+const config = require('../../../../../../config');
+
+const mockedFilePath = resolve(`${__dirname}/../../../../../../shared/dataService/business/network`);
+const dataServicePath = resolve(`${__dirname}/../../../../../../shared/dataService`);
+const mockedRequestFilePath = resolve(`${__dirname}/../../../../../../shared/utils/request`);
+
+const { mockChannelInfo } = require('../../../constants/transactionEstimateFees');
+
+beforeEach(() => {
+	jest.resetModules();
+
+	jest.mock('lisk-service-framework', () => {
```

Comment:
I think we can declare this on top instead of executing it before every testcase.